src/ui/authenticationInApp/ViewAuthenticationSsoAccount.qml

```diff
@@ -0,0 +1,70 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
```

Comment:
Doesn't look great on taller screens 
<img src="https://github.com/mozilla-mobile/mozilla-vpn-client/assets/15353801/7d6374c0-008f-4414-872e-db150ba87893"  width="50%" height="50%">

Reply:
there also seems to be some inconsistencies in the mocks themselves. sometimes this screen has 2 steps, other times it has 3 🤔

Reply:
This has been fixed (thanks @MattLichtenstein for all the QML assistance!)